.coveragerc

```diff
@@ -6,6 +6,7 @@ source =
 [paths]
 source =
     src/pyramid
+    */src/pyramid
```

Comment:
Is there an environment that does not find src files without this line? I get the same result when toggling between un/commenting this line on macOS.

Reply:
I don't remember why I have this in all of the `.coveragerc` files, it's something that in the past I've found makes coverage report work correctly.

Reply:
I think this is needed to report proper source paths even when tests are run against the package in tox’ testenv’s site-packages.